src/components/common/formFields/Checkbox.tsx

```diff
@@ -4,7 +4,7 @@ export enum CHECKBOX_VALUE {
     CHECKED = 'CHECKED',
     INTERMEDIATE = 'INTERMEDIATE',
 }
-interface CheckboxProps {
+export interface CheckboxProps {
```

Comment:
Can we abstract it out in type file?

Reply:
Yup. It'll be covered in one of the standardization tasks. As of now just exported it.